[16.0][ADD] partner_country_state_required (#1582)
OCA/partner-contact: partner_country_state_required/readme/DESCRIPTION.rst

> This module forces partners assigned to countries having fed. states should have a state assigned.

Reviewer: ```suggestion
This module makes the country state required on partners assigned to countries having fed. states.
```

Author: Applied!
